nntrainer/layers/cl_layers/addition_layer_cl.h

```diff
@@ -93,10 +94,20 @@ class AdditionLayerCL : public Layer {
    */
   const std::string getType() const override { return AdditionLayerCL::type; };
 
-  std::tuple<props::Print>
-    add_props; /**< fc layer properties : unit - number of output neurons */
+  /**
+   * @brief     registerClKernels for addition_layer_cl
+   * @details   registerClKernels for addition_layer_cl always returns true
+   * without any specific action for kernel registeration. It only uses
+   * cl_blas_kernels and there is no specific kernels for this. If there are
+   * specific kernels for this, it should be updated to register the kernels .
+   */
+  static bool registerClKernels() { return true; };
```

Comment:
does it register ClKernels?
or
does this check if CLKernels are registered or not?

(Clareify at brief)